src/artemis/experiment_plans/experiment_registry.py

```diff
@@ -22,24 +22,24 @@ def do_nothing():
     pass
 
 
-EXPERIMENT_TYPES = Union[GridScanParams, RotationScanParams]
-PLAN_REGISTRY: Dict[str, Dict[str, Callable]] = {
-    "fast_grid_scan": {
-        "setup": fast_grid_scan_plan.create_devices,
-        "run": fast_grid_scan_plan.get_plan,
-        "internal_param_type": FGSInternalParameters,
-        "experiment_param_type": GridScanParams,
-    },
-    "rotation_scan": {
-        "setup": do_nothing,
-        "run": not_implemented,
-        "internal_param_type": RotationInternalParameters,
-        "experiment_param_type": RotationScanParams,
-    },
-}
-EXPERIMENT_NAMES = list(PLAN_REGISTRY.keys())
-EXPERIMENT_TYPE_LIST = [p["experiment_param_type"] for p in PLAN_REGISTRY.values()]
-EXPERIMENT_TYPE_DICT = dict(zip(EXPERIMENT_NAMES, EXPERIMENT_TYPE_LIST))
+# EXPERIMENT_TYPES = Union[GridScanParams, RotationScanParams]
```

Comment:
Possibly everything in this module should go in `__main__`, since it's just the error type left

Reply:
`main` now has multiple plans listed here. We will need to run all of them

Reply:
Added `full_grid_scan` too